packages/cactus-plugin-ledger-connector-iroha2/README.md

```diff
@@ -0,0 +1,348 @@
+# `@hyperledger/cactus-plugin-ledger-connector-iroha2`
+
+This plugin provides `Cactus` a way to interact with **Iroha V2** networks. Using this we can run various Iroha leger instructions and queries.
+If you want to connect to Iroha V1 ledger, please use `@hyperledger/cactus-plugin-ledger-connector-iroha` instead.
+
+## Summary
+
+- [Remarks](#remarks)
+- [Getting Started](#getting-started)
+- [Endpoints](#endpoints)
+- [Running the tests](#running-the-tests)
+- [Contributing](#contributing)
+- [License](#license)
+- [Acknowledgments](#acknowledgments)
+
+## Remarks
+
+- Docker support is not implemented yet.
+- There is no official Iroha V2 release yet. API and connector behavior can change before stable release.
+- Query pagination is not supported yet. Querying large datasets (the ones with `All`) can be catastrophic. Pagination is not implemented in upstream javascript iroha sdk yet.
+- There's no way to check transaction status that we send. We're waiting for upstream change, see: https://github.com/hyperledger/iroha-javascript/issues/117
+- Connector can't be used with cactus module system yet. See: https://github.com/hyperledger/iroha-javascript/issues/111
+- Signing transactions on the client side is not supported yet, we're waiting for upstream change, see: https://github.com/hyperledger/iroha-javascript/issues/113
```

Comment:
This description seems old

Reply:
Please review only the last commit here, base iroha2 connector commit is not updated with original PR. Will do the rebase after irohav2 PR is merged.